### PR TITLE
adds support for sitelinks

### LINKED ISSUE #2: Access to sitelinks & parsing P1082
Hey!
I'm currently playing around with https://github.com/p-e-w/wd2sql which is using this library, and I'm trying to extract site links as well. If I'm not mistaken, that's completely unimplemented here, right? Sorry for asking dumb questions maybe, I just today started with Rust, so maybe I'm just overlooking something :sweat_smile: !?

I also had some trouble accessing [P1082](https://www.wikidata.org/wiki/Property:P1082) claims, which seem to have an incompatible format somehow; workaround for now is to ignore those claims

Would you be willing to release a new version if I make changes & provide a PR?

Cheers,
Lena

### PR DESCRIPTION
**Pull request by lenalebt**
This is my first rust PR, so please be patient if I forgot something :sweat_smile: . It's currently parsing a whole wikidata dump locally. So far ~75GiB (4.5 million entries) without an error.

P1082 parses fine with the most recent `master`, it was the released 0.3.0 that apparently had troubles with it.

Closes #2

### CODE REVIEW

File: src/entity.rs [hunk 1]
```diff
@@ -184,6 +186,22 @@ pub struct ClaimValue {
...
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+pub struct SiteName(pub String);
+
+/// A sitelink value.
+#[derive(Debug, Clone, PartialEq, Default, Serialize, Deserialize)]
+pub struct SitelinkValue {
+    /// The title of the site link.
+    pub title: String,
+    /// The badges of the article.
+    pub badges: Vec<WikiId>,
```
Comment (syvb) [inline src/entity.rs:200]: This should be a `Vec<Qid>`, since badges are always items (and not lexemes or properties).

**Review by syvb — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a few minor nits.

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wikidata"
-version = "0.3.0"
+version = "0.4.0"
```
[the remark this answers is not in the document]
Comment (lenalebt) [reply Cargo.toml]: aye, changed it :-)

File: src/entity.rs (hunk 1 above)
Comment (lenalebt) [reply src/entity.rs]: Okay, I changed it! Using this lib to get into the details of wikidata, good that you already know more of the details :-)

**Review by syvb — APPROVED** [review]
Thanks for the PR! I just pushed a commit to run `cargo fmt` for style consistency.

I'll release 0.3.1 shortly.

**Comment by syvb** [comment]
0.3.1 is now released.

**Pull request merged**